samples/client/petstore/java/webclient-jakarta/src/main/java/org/openapitools/client/ApiClient.java

```diff
@@ -152,7 +152,6 @@ protected void init() {
         authentications.put("api_key_query", new ApiKeyAuth("query", "api_key_query"));
         authentications.put("http_basic_test", new HttpBasicAuth());
         authentications.put("bearer_test", new HttpBearerAuth("bearer"));
-        authentications.put("http_signature_test", new HttpBearerAuth("signature"));
```

Comment:
note for discussion: when http_signature_test is kept, the samples contain this incorrect line of code

Reply:
outdated now